exporter/awss3exporter/README.md

```diff
@@ -42,6 +42,8 @@ Marshaler determines the format of data sent to AWS S3. Currently, the following
 - `otlp_json` (default): the [OpenTelemetry Protocol format](https://github.com/open-telemetry/opentelemetry-proto), represented as json.
 - `sumo_ic`: the [Sumo Logic Installed Collector Archive format](https://help.sumologic.com/docs/manage/data-archiving/archive/).
   **This format is supported only for logs.**
+- `body_log`: If you specifically intended to export only the body of logs and ignore other labels.
```

Comment:
```suggestion
- `body_log`: export the log body as string
```
Can we change from `body_log` to `body`?

Reply:
Yes, I can. It's changed to "body" in the latest commit.